content/en/references/ibd-system-electronic-sensing-system-dataflow.adoc

```diff
@@ -0,0 +1,12 @@
+---
+title: "Sensing System UML Diagram"
+linkTitle: "Sensing System UML Diagram"
+weight: 1
+description: >
+  UML diagram of Project Tetra's electronic sensing system.
```

Comment:
I would re-title the description to _Interface control document for electronic sensing system_